tests/printer/expr/expected/asyncAwait.res.txt

```diff
@@ -106,7 +106,7 @@ let _ = await {
 
 let f1 = async (~x, ~y) => x + y
 let f2 = async (@foo ~x, @bar ~y) => x + y
-let f3 = async (@bar @foo ~x as @zz z, ~y) => x + y
+let f3 = @foo async (~x as @bar @zz z, ~y) => x + y
```

Comment:
I'm not sure about this expression `@attr1 ~x as @attr2 y`.
If we parse the attributes inside `(...)`, the attributes are going to be `ppat_attributes`, the printer emits `~x as @attr1 @attr2 y` What do you think?

Reply:
Looks fine. There's only one place for attributes on labels.